lib/utils.rb

```diff
@@ -49,6 +49,40 @@ def self.longest_common_call_number_prefix(*strs)
     substr.sub(/(\s|[[:punct:]])+\z/, '')
   end
 
+  # Extract century and decade from year.
+  # "Obsolete. to take a tenth of or from." - https://www.dictionary.com/browse/decimate
+  #
+  # @param maybe_year [Integer] an int that (hopefully) represents a year
+  def self.centimate_and_decimate(maybe_year)
```

Comment:
alternatively, i could pick a less precious and slightly wordier name, like `extract_century_and_year_strings`